src/ml_tooling/plots/precesion_recall_curve.py

```diff
@@ -0,0 +1,44 @@
+from matplotlib import pyplot as plt
+from matplotlib.axes import Axes
+from sklearn.metrics import precision_recall_curve, average_precision_score
+
+from ml_tooling.utils import DataType
+
+
+def plot_pr_curve(y_true: DataType, y_proba: DataType, title: str = None, ax: Axes = None) -> Axes:
+    """
+    Plot precsion-recall curve. Works only with probabilities
+
+    :param y_true:
```

Comment:
Can we change this to use Numpy style? 
Still haven't converted everything, but working on it as part of the docs branch